src/components/left-menu/LeftMenu.js

```diff
@@ -24,6 +24,7 @@ class LeftMenu extends Component {
                 <button onClick={() => this.handleToolSelect("selectTool")}>
                     <img src="./assets/001-cursor.svg" alt="select" id="button-icon" />
                 </button>
+                <button onClick={() => this.handleToolSelect("panTool")} id="temp">p</button>
```

Comment:
should id="temp" be changed or removed?

Reply:
@VLuisa is going to pull from master and fix up these temp buttons I created once we've merged (rather than pulling directly from my branch)